bvm/ballerina-core/src/main/java/org/ballerinalang/BLangProgramRunner.java

```diff
@@ -30,13 +26,17 @@
 import org.ballerinalang.util.exceptions.BallerinaException;
 import org.ballerinalang.util.program.BLangFunctions;
 
+import static org.ballerinalang.util.cli.ArgumentParser.extractEntryFuncArgs;
 /**
  * This class contains utilities to execute Ballerina main and service programs.
  *
  * @since 0.8.0
  */
 public class BLangProgramRunner {
 
+    public static final String MAIN = "main";
```

Comment:
Shall we use BLangConstants.MAIN_FUNCTION_NAME instead creating a new const here